redhat-appstudio/service-provider-integration-oauth
# kcp manual token upload

### What does this PR do?
endpoint for manual token upload with kcp workspace

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-199

### How to test this PR?

`quay.io/mvala/spi-oauth:svpi138-kcpTokenUpload`

1. deploy to KCP
2. create SPIAccessToken
3. upload token to URL in `status.uploadURL`
4. you should get `204` response and token should be stored in Vault (the status of the SPIAccessToken won't be updated, will update in followup PR)
